### DISCUSSION TITLE
Closing Upload window

### DISCUSSION THREAD

**Discussion by THF-Mediapics** [photoprism/photoprism#4652] [Q&A]
I cannot close the upload window although photoprism is already indexing. A message is displayed at the bottom of the screen saying “Images are being uploaded”. To continue working, simply call up photoprism again and can then continue working there as normal. I cannot understand why the upload window cannot be closed after uploading.

![Bildschirmfoto 2024-12-04 um 14 12 55](https://github.com/user-attachments/assets/1d203108-deb4-4b2e-8e05-705349d74b7e)

**Comment by graciousgrey** [comment]
Thank you very much for your feedback! This should be improved with this:

- https://github.com/photoprism/photoprism/issues/4799
